annotator/static/views/player.js

```diff
@@ -46,6 +46,12 @@ class PlayerView {
         // Video URL
         this.videoSrc = videoSrc;
 
+        // Video start time
+        this.videoStart = parseFloat(videoStart);
```

Comment:
in views.py, you should just have `float(request.GET.get('s', <something>))`. It should already be inside a `json.dumps` call